mm2src/mm2_main/src/rpc.rs

```diff
@@ -425,6 +426,11 @@ pub extern "C" fn spawn_rpc(ctx_h: u32) {
     // By entering the context, we tie `tokio::spawn` to this executor.
     let _runtime_guard = CORE.0.enter();
 
+    // Create a TcpListener
+    // Binds on the specified IP and port or allocates a random port if the port is 0.
+    let listener =
+        TcpListener::bind(rpc_ip_port).unwrap_or_else(|err| panic!("Can't bind on {}: {}", rpc_ip_port, err));
+
```

Comment:
it will be useful if we log the listening address. I think we use too before but it just disappeared

Reply:
`😉 2025-02-07 15:38:40 -0500 [] rpc:471] >>>>>>>>>> DEX stats 127.0.0.1:60432 DEX stats API enabled at unixtime.1738960720 <<<<<<<<<`

It will be displayed like this. Do you want an additional message?

Reply:
ah looks good!